components/image/Image.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import { SHOW_IMAGES } from 'proton-shared/lib/constants';
+import Button from '../button/Button';
+import useMailSettings from '../../hooks/useMailSettings';
+
+const Image = (props) => {
```

Comment:
Since this component used `useMailSettings` hook, it should not be defined as generic component.